File: golden-square-solo-project/lib/delivery_text_formatter.rb
class DeliveryTextFormatter
  def initialize(order, delivery_time = DeliveryTime.new)
    @delivery_time = delivery_time.get_delivery_time
    @order = order
  end

  def format
    message = 'Thank you! Your order was placed and will be delivered before '
    message += "#{(@order.get_order_time + @delivery_time*60).strftime("%k:%M")}"
  end
end

File: java-takeaway-challenge/app/src/main/java/OrderSystem/DeliveryTextFormatter.java
package OrderSystem;

import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class DeliveryTextFormatter {
  LocalDateTime time;

  public DeliveryTextFormatter(LocalDateTime time) {
    this.time = time;
  }

  public String renderText() {
    DateTimeFormatter timeFormatter = DateTimeFormatter.ofPattern("HH:mm");
    String text;

    LocalDateTime deliveryTime = time.plusHours(1);
    text = String.format("Thank you! Your order was placed and will be delivered before %s",
        deliveryTime.format(timeFormatter));
    return text;
  }

}
